packages/g/gammaray/package.yml

```diff
@@ -0,0 +1,85 @@
+name       : gammaray
+version    : 3.0.0
+release    : 1
+source     :
+    - https://github.com/KDAB/GammaRay/releases/download/v3.0.0/gammaray-3.0.0.tar.gz: acd27dbbcbdf73fed497e0b5d6c477f2e11b59c48499752602677037dcd64ba5
+homepage   : https://www.kdab.com/development-resources/qt-tools/gammaray/
+license    : GPL-2.0-or-later
+component  :
+    - programming.tools
+    - probe-qt5 : programming.tools
+    - probe-qt5-devel : programming.devel
+summary    :
+    - GammaRay is a tool to poke around in a Qt-application
+    - probe-qt5 : Qt 5 probe for GammaRay
+    - probe-qt5-devel : Development files for gammaray-probe-qt5
+description:
+    - GammaRay is a software introspection tool for Qt applications developed by KDAB. Leveraging the QObject introspection mechanism it allows you to observe and manipulate your application at runtime. This works both locally on your workstation and remotely on an embedded target.
+    - probe-qt5 : Provides a Qt 5 probe for GammaRay that allows introspecting Qt 5 applications.
+    - probe-qt5-devel : Provides a Qt 5 probe for GammaRay that allows introspecting Qt 5 applications.
+clang      : yes
+optimize   : lto
+builddeps  :
+    - pkgconfig(Qt5Core)
+    - pkgconfig(Qt5Quick)
+    - pkgconfig(Qt5Script)
+    - pkgconfig(Qt5Scxml)
+    - pkgconfig(Qt5Svg)
+    - pkgconfig(Qt5WaylandCompositor)
+    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt63DAnimation)
+    - pkgconfig(Qt6Bluetooth)
+    - pkgconfig(Qt6Concurrent)
+    - pkgconfig(Qt6Core)
+    - pkgconfig(Qt6Designer)
+    - pkgconfig(Qt6Location)
+    - pkgconfig(Qt6Positioning)
+    - pkgconfig(Qt6Qml)
+    - pkgconfig(Qt6Scxml)
+    - pkgconfig(Qt6ShaderTools)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6WaylandCompositor)
+    - pkgconfig(Qt6WebEngineWidgets)
+    - kf6-syntax-highlighting-devel
+    - qt6-doctools
+rundeps    :
+    - lldb
+setup      : |
+    # Fix plugin install dir
+    sed -e 's|plugins/gammaray|lib/qt6/plugins/gammaray|' -i CMakeLists.txt
+
+    %patch -p1 -i $pkgfiles/gammaray-fix-build-with-qt-6.7.patch
+
+    # Build the Qt6 client and probe
+    %cmake_ninja -B qt6-build \
+        -DQT_VERSION_MAJOR=6 \
+        -DGAMMARAY_QT6_BUILD:BOOL=TRUE \
+        -DGAMMARAY_BUILD_DOCS:BOOL=TRUE \
+        -DGAMMARAY_INSTALL_QT_LAYOUT:BOOL=FALSE \
+        -DECM_MKSPECS_INSTALL_DIR:PATH=%libdir%/qt6/mkspecs/modules \
+        -DPLUGIN_INSTALL_DIR:PATH=%libdir%/qt6/plugins/gammaray
+
+    # Build the Qt5 probe only
+    %cmake_ninja -B qt5-build \
+        -DQT_VERSION_MAJOR=5 \
+        -DGAMMARAY_QT6_BUILD:BOOL=FALSE \
+        -DGAMMARAY_BUILD_UI:BOOL=FALSE \
+        -DGAMMARAY_BUILD_DOCS:BOOL=FALSE \
+        -DGAMMARAY_PROBE_ONLY_BUILD:BOOL=TRUE \
+        -DGAMMARAY_INSTALL_QT_LAYOUT:BOOL=FALSE
+build      : |
+    %ninja_build -C qt6-build
+    %ninja_build -C qt5-build
```

Comment:
Swap the order of the Qt5 and Qt6 builds and installs. When you install from multiple builds like this usually the second build will overwrite any files from the first build if they install duplicate files. Since Qt6 is our preferred build we want it to override any Qt5 conflicts and not the other way around.